gridt/src/app/core/api.service.ts

```diff
@@ -230,17 +249,6 @@ export class ApiService {
     );
   }
 
-  /**
-  * Observable to obtain identity from server.
-  */
-  public userIdentity$: Observable<Identity> = this.auth.readyAuthentication$.pipe(
-    flatMap((options) => this.http.get<Identity>(
-      `${this.URL}/identity`,
-      options
-    )),
-    catchError( this.handleBadAuth())
-  );
-
```

Comment:
this was changed due to my linter giving this:
Declaration of instance field not allowed after declaration of instance method. Instead, this should come at the beginning of the class/interface. (member-ordering)tslint(1)